docs/cli.md

```diff
@@ -1,70 +1,77 @@
 # Command Line Interface (CLI)
 
-**Pydantic AI** comes with a CLI, `clai` (pronounced "clay") which you can use to interact with various LLMs from the command line.
-It provides a convenient way to chat with language models and quickly get answers right in the terminal.
+**Pydantic AI** comes with a CLI, `clai` (pronounced "clay"). You can use it to chat with various LLMs and quickly get answers, right from the command line, or spin up a uvicorn server to chat with your Pydantic AI agents from your browser.
 
 We originally developed this CLI for our own use, but found ourselves using it so frequently that we decided to share it as part of the Pydantic AI package.
 
 We plan to continue adding new features, such as interaction with MCP servers, access to tools, and more.
 
-## Usage
+## Installation
 
-<!-- Keep this in sync with clai/README.md -->
-
-You'll need to set an environment variable depending on the provider you intend to use.
-
-E.g. if you're using OpenAI, set the `OPENAI_API_KEY` environment variable:
-
-```bash
-export OPENAI_API_KEY='your-api-key-here'
-```
-
-Then with [`uvx`](https://docs.astral.sh/uv/guides/tools/), run:
+You can run the `clai` using [`uvx`](https://docs.astral.sh/uv/guides/tools/):
```

Comment:
Does this also install the packages necessary for `clai web`? I think it should, or at least we should have an optional group `clai[web]` that we can tell the user to install from the error message

Reply:
not sure how to make a test that tests this, but I tested in locally by running from /tmp and it worked. I think though that the deps are installed anyway via fastmcp and mcp.